dashboard/src/index.html

```diff
@@ -11,7 +11,7 @@
 
 -->
 <!doctype html>
-<html class="no-js" ng-app="userDashboard" ng-strict-di>
+<html class="no-js">
```

Comment:
we don't need anymore ng-app ?

Reply:
@davidfestal can you comment. Why it was removed? https://github.com/redhat-developer/rh-che/blame/master/assembly/dashboard/src/main/patches/src/index.html.patch#L7 ?

Reply:
It was like that from the beginning of the Keycloak support in the dashboard (initial draft was made by @gorkem).
I think the reason is that angular is now bootstrapped *after* the keycloak initialization, as shown here: https://github.com/redhat-developer/rh-che/blob/master/assembly/dashboard/src/main/patches/src/app/index.module.ts.patch#L31

Reply:
@davidfestal @benoitf @ashumilova @akurinnoy do you see any blockers if we revert this change?

Reply:
I don't know if keycloak need to be initialized before but I'm fine in both cases. But it would be great to not have this lazy loading in index.module.ts

Reply:
why do you want to revert it ?
it is consistent with the fact that angular is bootstrapped after keycloak initialization also here: https://github.com/eclipse/che/pull/6441/files#diff-23910a1448372bee6229c74377877dfdR112

I'm not sure it would work if you revert this change.

Reply:
it was part of the design of the keycloak integration to the dashboard from the very beginning. @gorkem wdyt ?

Reply:
Ok. Based on this https://github.com/eclipse/che/pull/6441#discussion_r141333945 and if there are no major objections let's keep it.

Reply:
IMHO I still think we don't require to do a manual initialization of angular